src/contractTest/java/uk/gov/hmcts/reform/wataskmanagementapi/provider/TaskManagementGetTaskBySearchForCompletablePactTest.java

```diff
@@ -131,6 +131,7 @@ public List<Task> createTasks() {
             new WarningValues(Collections.emptyList()),
             "Case Management Category",
             "hearing_work",
+            "Hearing work",
```

Comment:
could we also update the consumer (TaskManagementGetTasksBySearchForCompletableConsumerTest) to now include the workType label